website/bin/kill_server.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+tmux kill-session -t "snack"
```

Comment:
This will be removed and replaced by tooling for easily starting snack services locally